desktop/menus/edit-menu.js

```diff
@@ -50,6 +51,13 @@ const buildEditMenu = settings => {
         checked: settings.spellCheckEnabled,
         click: appCommandSender({ action: 'toggleSpellCheck' }),
       },
+      {
+        label: '&Auto-Detect Language',
+        type: 'checkbox',
+        checked: settings.languageDetectionEnabled,
+        click: appCommandSender({ action: 'toggleLanguageDetection' }),
+        visible: !platform.isOSX(), // currently not working on Mac
+      },
```

Comment:
Besides disabling language-detection (and switching back to English?) is there any way to manually choose the language?

I can imagine that if someone wants to write in one language about another language the detection could be consistently wrong but then I'd want to manually fix it.

For example, writing in German about French poetry.

Reply:
It is technically feasible, but I think that use case probably isn't common enough to warrant the clutter of an additional menu item. However, I did initially post a UI suggestion for a global manual select (as well as some other, per-note possibilities) [in the original issue](https://github.com/Automattic/simplenote-electron/issues/949#issuecomment-435934703).

This is mostly a UI design decision, so my thinking was to start out with the most simplest design, which is auto detection. If that works well enough for the majority of our users, great. If not, we can revisit.

Reply:
> I did initially post a UI suggestion for a global manual select

Yeah that's what I meant. It's a good idea.

> start out with the most simplest design

Also a great idea. I wouldn't want to hold up this PR for not adding more but I wanted to ask if it was something already available we might have just been overlooking, for example, in the right-click menu of the text view if all we had to do were set some boolean toggle…

Reply:
> I wanted to ask if it was something already available we might have just been overlooking

I'm afraid not 😅 Also, I wrote "technically feasible" but I overlooked one possible complication we might run into: Getting a definitive list of available spell checker languages. I haven't looked into it too deeply, but it doesn't seem like a language list is exposed through `electron-spellchecker`. If Chromium doesn't expose a list, it might get complicated. (Just a heads up in case you consider it in the future.)